config/scanner.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
```

Comment:
Datei von `config/scanner.php` in `config/crawler.php` oder `config/siwecos.php` umbenennen.